### PR TITLE
Add management command to get couch cases non updated in ES

### PR DESCRIPTION
**Pull request by sravfeyn**
https://dimagi-dev.atlassian.net/browse/HI-835

@dannyroberts

### CODE REVIEW

File: corehq/apps/hqadmin/management/commands/stale_cases_in_es.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        include_docs=False,
+        reduce=False
+    ), chunk_size)
+    for chunk in chunked_iterator:
+        case_ids = [row['id'] for row in chunk]
+        results = (CaseES(es_instance_alias=ES_EXPORT_INSTANCE)
+            .domain(domain)
+            .case_ids(case_ids)
+            .values_list('_id', 'server_modified_on'))
+        es_modified_on_by_ids = {_id: modified_on for _id, modified_on in results }
```
Comment (czue) [inline corehq/apps/hqadmin/management/commands/stale_cases_in_es.py]: If I'm reading this right, I think this line could be replaced by `dict(results)`

File: corehq/apps/hqadmin/management/commands/stale_cases_in_es.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    """
+    help = inspect.cleandoc(__doc__).split('\n')[0]
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+
+    def handle(self, domain, **options):
+        for case_id in get_server_modified_on_for_domain(domain):
+            print(case_id)
```
Comment (czue) [inline corehq/apps/hqadmin/management/commands/stale_cases_in_es.py]: nit: these aren't `case_ids` but three-ples, no?

**Review by czue — APPROVED** [review]
nothing blocking

File: corehq/apps/hqadmin/management/commands/stale_cases_in_es.py (hunk 1 above)
Comment (sravfeyn) [reply corehq/apps/hqadmin/management/commands/stale_cases_in_es.py]: True

File: corehq/apps/hqadmin/management/commands/stale_cases_in_es.py (hunk 2 above)
Comment (sravfeyn) [reply corehq/apps/hqadmin/management/commands/stale_cases_in_es.py]: Yes, meant to print as CSV. Will correct, thanks

**Review by dannyroberts — APPROVED** [review]
Looks good! Thanks for putting this together. I'm pretty interested to see what the results are

**Comment by sravfeyn** [comment]
Added one more management command to reprocess the cases

**Comment by sravfeyn** [comment]
Danny, I have added the result file and relevant discussion in the ticket here https://dimagi-dev.atlassian.net/browse/HI-835

File: corehq/apps/hqadmin/management/commands/republish_couch_case_changes.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        self.publish_cases(domain, case_ids)
+
+    def get_case_ids(self, case_ids_file):
+        with open(case_ids_file, 'r') as f:
+            lines = f.readlines()
+            return [l.strip() for l in lines]
+
+    def publish_cases(self, domain, case_ids):
+        from corehq.apps.hqcase.management.commands.backfill_couch_forms_and_cases import (
+            publish_change, create_case_change_meta)
```
Comment (czue) [inline corehq/apps/hqadmin/management/commands/republish_couch_case_changes.py]: nit: could consider pulling these out into a utility module if they're going to be used outside the defining command.

File: corehq/apps/hqadmin/management/commands/republish_couch_case_changes.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+from django.core.management import BaseCommand
+
+from dimagi.utils.chunked import chunked
+
+from casexml.apps.case.models import CommCareCase
+from corehq.doctypemigrations.continuous_migrate import _bulk_get_revs
```
Comment (czue) [inline corehq/apps/hqadmin/management/commands/republish_couch_case_changes.py]: should this be renamed to `bulk_get_revs` if it is used in other places?

**Review by czue — APPROVED** [review]
still nothing blocking

File: corehq/apps/hqadmin/management/commands/republish_couch_case_changes.py (hunk 4 above)
Comment (sravfeyn) [reply corehq/apps/hqadmin/management/commands/republish_couch_case_changes.py]: Yes, updated.

**Comment by czue** [comment]
@sravfeyn any reason this never got merged?

**Comment by sravfeyn** [comment]
Not really! Merging it now.

**Pull request merged**